src/lib/helpers/test/setupTestEnv.ts

```diff
@@ -0,0 +1,44 @@
+import { createStealthClient } from '../..';
+import { ERC5564_CONTRACT } from '../../..';
+import type { VALID_CHAIN_IDS } from '../types';
+
+function setupTestEnv() {
+  if (!process.env.TEST_CHAIN_ID) {
```

Comment:
Let's create `.env.example` files for the env vars in this file.

Reply:
[a5b7541](https://github.com/ScopeLift/stealth-address-sdk/pull/15/commits/a5b7541b0be7de96edf9735392166356ce09eeda)